alerts/cluster-monitoring-operator/PrometheusRuleFailures.md

```diff
@@ -0,0 +1,36 @@
+# PrometheusRuleFailures
+
+## Meaning
+
+The `PrometheusRuleFailures` alert triggers when there has been
+a constant increase in failed evaluations of prometheus rules for more 
+than 15 minutes.
+
+## Impact
+
+Metrics and alerts might be missing or inaccurate.
+
+## Diagnosis
+
+1. Determine whether the alert has triggered for the instance of Prometheus used
+for default cluster monitoring or for the instance that monitors user-defined
+projects by viewing the alert message's `namespace` label: the namespace for
+default cluster monitoring is `openshift-monitoring`; the namespace for user
+workload monitoring is `openshift-user-workload-monitoring`.
+
+1. Review the logs for the affected Prometheus instance:
+
+    ```console
+    $ NAMESPACE='<value of namespace label from alert>'
+
+    $ oc -n $NAMESPACE logs -l 'app=prometheus' | grep -o 'Evaluating rule failed.*' | sort | uniq -c | sort -n
+    level=error ... msg="Evaluating rule failed." ...
+    ```
+
+1. It's also possible to evaluate the rule expression using the OpenShift UI, or maybe using PromLens to help debug expression issues.
+
+## Mitigation
+
+If the logs indicate a syntax or other configuration error, correct the
```

Comment:
it might not be possible for the user to fix the error if the rule is shipped by an OpenShift component. In this case, the user needs to open a support case and eventually the buggy expression will be fixed.